tests/components/google_sheets/test_init.py

```diff
@@ -77,12 +78,12 @@ async def func() -> None:
 
     # Verify clean unload
     entries = hass.config_entries.async_entries(DOMAIN)
-    assert len(entries) == 1
-    await hass.config_entries.async_unload(entries[0].entry_id)
-    await hass.async_block_till_done()
+    for entry in entries:
+        await hass.config_entries.async_unload(entry.entry_id)
```

Comment:
Why do we need to do this here in the fixture? Isn't enough to do it explicitly in one test and assert the result?

Reply:
It's overkill. Fixed.

I did this in nest, but maybe it was necessary there given all the possible states of the integration (sometimes it does cleanup on unload, sometimes not). Removed.